Test/test.js

```diff
@@ -1,17 +1,15 @@
-const mongoose = require('mongoose');
-const database = require('../database/db.js');
+// const mongoose = require('mongoose');
+const database = require('../database/db_pg.js');
 const chai = require('chai');
 const chaiHttp = require('chai-http');
-const app = require('../server/server.js');
+const app = require('../server/server_pg.js');
```

Comment:
As this gets busier switching packages back and forth, consider splitting out your tests into multiple files in the Test folder.